andela/workdey
# Ft payment solution #123033561

# #123033561
#### What does this PR do?

This **PR** serves to include the functionality to have to pay before getting a task done.
#### Description of Task to be completed?

When a tasker creates/assigns a task, a taskee should not see the notification unless the tasker has paid. This payment functionality is handled by **Stripe**.
#### How should this be manually tested?

Go to the homepage of the app.
- Search for taskees with a particular skillset.
- Select a taskee from the given result set.
- Assign the task
- Pay with Stripe
- You will be redirected to the dashboard with a confirmation message
- You may want to log in to your account on Stripe to see the payments.
#### Any background context you want to provide?

Before now, a taskee gets a notification about a newly assigned task and could go ahead to carry out the tasks even if the tasker has not paid **_Workdey Inc**_
#### What are the relevant pivotal tracker stories?

[#123033561]
#### Screenshots (if appropriate)

![screen shot 2016-07-20 at 3 52 01 pm](https://cloud.githubusercontent.com/assets/14159416/16991035/2841607c-4e92-11e6-8fa7-8debc162e035.png)

![screen shot 2016-07-20 at 3 52 22 pm](https://cloud.githubusercontent.com/assets/14159416/16991058/3c1c8766-4e92-11e6-866c-283fd0fe10c7.png)
#### Questions:
